start

```diff
@@ -21,6 +24,10 @@ function main() {
 	echo ""
 
 	process_args $*
+	if [ "$NETWORK" != "standalone" ] && [ "$ENABLE_HORIZON_CAPTIVE_CORE" = "true" ]; then
+	  echo "--enable-horizon-captive-core is only supported in the standalone network" >&2
+		exit 1
+	fi
```

Comment:
Shouldn't we have the same check for `ENABLE_CORE_MANUAL_CLOSE`?

Reply:
No, that should work for all networks